src/simularium/PDBModel.ts

```diff
@@ -74,24 +73,24 @@ class PDBModel {
     private bounds: Box3;
     // cancelled means we have abandoned this pdb
     // and if it is still initializing, it should be dropped/ignored as soon as processing is done
-    private cancelled: boolean;
+    private _cancelled: boolean;
```

Comment:
made this change so I could call `geometry.cancelled` for either geometry type and it would work.